drivers: usb_dc_stm32 broken after west update

after "west update" the stm32 usb driver is broken
```
[ 80%] Built target drivers__i2c
/home/stefan/zephyr/drivers/usb/device/usb_dc_stm32.c: In function 'usb_dc_stm32_get_maximum_speed':
/home/stefan/zephyr/drivers/usb/device/usb_dc_stm32.c:326:11: error: 'USB_OTG_SPEED_LOW' undeclared (first use in this function); did you mean 'USB_OTG_SPEED_FULL'?
   speed = USB_OTG_SPEED_LOW;
           ^~~~~~~~~~~~~~~~~
           USB_OTG_SPEED_FULL
/home/stefan/zephyr/drivers/usb/device/usb_dc_stm32.c:326:11: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [zephyr/CMakeFiles/zephyr.dir/build.make:518: zephyr/CMakeFiles/zephyr.dir/drivers/usb/device/usb_dc_stm32.c.obj] Error 1

```

--- a/drivers/usb/device/usb_dc_stm32.c
+++ b/drivers/usb/device/usb_dc_stm32.c
@@ -322,8 +322,6 @@ static u32_t usb_dc_stm32_get_maximum_speed(void)
 #else
 		speed = USB_OTG_SPEED_FULL;
 #endif /* DT_COMPAT_ST_STM32_USBPHYC && DT_USB_HS_BASE_ADDRESS */
-	} else if (!strncmp(DT_USB_MAXIMUM_SPEED, "low-speed", 9)) {
-		speed = USB_OTG_SPEED_LOW;
 	} else {
 		LOG_DBG("Unsupported maximum speed defined in device tree. "
 			"USB controller will default to its maximum HW "

